drivers/unix/os_unix.h

```diff
@@ -101,6 +103,10 @@ class OS_Unix : public OS {
 
 	virtual String get_executable_path() const override;
 	virtual String get_user_data_dir() const override;
+
+#if !defined(__ANDROID__) && !defined(__EMSCRIPTEN__)
+	virtual Vector<StackInfo> get_cpp_stack_info() const override;
+#endif
```

Comment:
These should be in the respective OS implementations I'd say, i.e. Mac and Linux, or overridden as stubs in Android and Web, this makes it hard to track

Reply:
@AThousandShips I think it makes the most sense to keep them in `OS_Unix` because `execinfo.h` is theoretically a part of any standards-compliant Unix system. The problem is that Android and Emscripten aren't fully POSIX compliant and don't provide the `execinfo.h` header. I was surprised to even see that they were children of `OS_Unix` because neither are really Unix systems. I can remove the `#ifdef`s from the header here, but I will need to keep them in `os_unix.cpp` (which already has a ton of these because of the differences in different Unix implementations) because the implementation there depends on being able to import `execinfo.h`, which is not present in either

Reply:
Fair enough, but this makes it harder to find as it'd be expected to be in those platforms instead of drivers

Reply:
Yeah, it's weird to me that `OS_MacOS` in `platform` inherits from `OS_Unix` in `drivers`, which inherits from `OS` in `core`, and each provides different portions of the platform class's implementation, but here we are.